src/lips.js

```diff
@@ -3499,7 +3499,7 @@ function extract_patterns(pattern, code, symbols, ellipsis_symbol, scope = {}) {
     function log(x) {
         /* istanbul ignore next */
         if (is_debug()) {
-            console.log(x);
+            console.log(x); // @jcubic can you enter what this console log returns here
```

Comment:
This is another debug function but this time local to this part of the code. It prints whatever is passed to the log function inside the code. This is a complex part of the code that processes part of `syntax-rules` (hygienic macros). The code is not yet fully working, that's why there is this debug statement here. With `is_debug()` I can test the Scheme code and turn of debugging logging expressions.